lib/ansible/plugins/callback/logdna.py

```diff
@@ -0,0 +1,204 @@
+# (c) 2018, Samir Musali <samir.musali@logdna.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    callback: logdna
+    callback_type: aggregate
+    short_description: Sends playbook logs to LogDNA
+    description:
+      - This callback will report logs from playbook actions, tasks, and events to LogDNA (https://app.logdna.com)
+    version_added: 0.1
```

Comment:
The value of `version_added` should be the version of Ansible in which the feature is expected to be added.

Reply:
Done - added - running checks

Reply:
@mattclay , it's done! Finished running checks

Reply:
this would be '2.7' at this point